ROADMAP.md

```diff
@@ -0,0 +1,94 @@
+# io.js Roadmap
+
+***This is a living document, it describes our policy and priorities as they exist today but can evolve over time.***
+
+## Stability Policy
+
+The single more important consideration in every code change is the impact it will have, positive or negative, on the ecosystem (modules and applications).
+
+io.js does not remove JavaScript API.
```

Comment:
I would clarify this a bit. By JS API, do you mean "EcmaScript" API or node core API?